src/cli.js

```diff
@@ -1,3 +1,6 @@
+// Disable warnings about template strings in this file, as we do lots of
+// string concatenation so options/help in the CLI app display nicely.
+// jscs:disable requireTemplateStrings
```

Comment:
Arg, I just noticed what you mean about the whitespace. I'll leave this one out for now, it's not too bad as-is.